src/graphql/models/Article.js

```diff
@@ -40,22 +40,45 @@ const Article = new GraphQLObjectType({
     },
     articleReplies: {
       type: new GraphQLList(ArticleReply),
+      description:
+        'Connections between this article and replies. Sorted by the logic described in https://github.com/cofacts/rumors-line-bot/issues/78.',
       args: {
         status: {
           type: ArticleReplyStatusEnum,
           description:
             'When specified, returns only article replies with the specified status',
         },
       },
-      resolve: async ({ id, articleReplies = [] }, { status }) =>
-        filterArticleRepliesByStatus(
+      resolve: async ({ id, articleReplies = [] }, { status }) => {
+        const sortedArticleReplies = filterArticleRepliesByStatus(
           // Inject articleId to each articleReply
           articleReplies.map(articleReply => {
             articleReply.articleId = id;
             return articleReply;
           }),
           status
-        ),
+        ).sort(
+          (a, b) =>
+            b.positiveFeedbackCount -
+            b.negativeFeedbackCount -
+            (a.positiveFeedbackCount - a.negativeFeedbackCount)
+        );
+
+        if (articleReplies.length === 0) return [];
```

Comment:
Is it better to move this above line53?

Reply:
You found a bug.

`filterArticleRepliesByStatus` may reduce the number of articleReplies, I should use `sortedArticleReplies` instead of `articleReplies` 
 here.

Reply:
Fixed in 24a306340a4f58d0b11f97ba0564de02a574751e .
Thanks a lot for spotting this! @nonumpa